project/assets/asset_utils.ts

```diff
@@ -0,0 +1,14 @@
+import { AssetFont } from "./asset_font"
+import { AssetGlb } from "./asset_glb"
+import { AssetGodotScene } from "./asset_godot_scene"
+import { AssetImage } from "./asset_image"
+
+export function allAssetExtensions() {
+  return [
+    ...AssetFont.extensions(),
+    ...AssetImage.extensions(),
+    ...AssetGodotScene.extensions(),
+    ...AssetGlb.extensions(),
+    ".godot",
```

Comment:
Makes me think GodotProjectFile should extend BaseAsset someday. :)

Reply:
I actually have that as part of the windows mr (as I needed better shared path handling) :D